.github/workflows/deploy-containers.yml

```diff
@@ -97,7 +91,7 @@ jobs:
         with:
           task-definition: ${{ steps.taskdef.outputs.task-definition }}
           service: ${{ matrix.SERVICE_NAME }}
-          cluster: ${{ steps.cluster.outputs.name }}
+          cluster: ${{ secrets.STAGING_ECS_CLUSTER_NAME }}
```

Comment:
Why pull this from secret it's public info:
https://github.com/cds-snc/covid-alert-server-staging-terraform/blob/83cf1e2d95ada5aafe12411dd66bef08cb24463d/server/aws/variables.auto.tfvars#L20

We can just hardcode and remove a chance for an error.

Reply:
I'd put it as a secret to make it updatable without changing the workflow, but not a problem to just hard code it in the file.